kate/recovery/src/testnet.rs

```diff
@@ -0,0 +1,20 @@
+use std::{collections::HashMap, sync::Mutex};
+
+use dusk_plonk::commitment_scheme::kzg10::PublicParameters;
+use once_cell::sync::Lazy;
+use rand::SeedableRng;
+use rand_chacha::ChaChaRng;
+
+static SRS_DATA: Lazy<Mutex<HashMap<usize, PublicParameters>>> =
+	Lazy::new(|| Mutex::new(HashMap::new()));
+
+pub fn public_params(max_degree: usize) -> PublicParameters {
```

Comment:
As `max_degree` is always related with `columns` we should keep the previous strong type. Did I miss something?

Reply:
I'm not sure if that was optimization which couples us to number of columns per block, or we could go with some maximum value as on light-client (1024 is current value), but thats something someone needs to explore and confirm. 

However, main reason why strong type is removed is that intention was to remove duplication (there were two fns for creating public_parameters), and to be able to use it on light client also, which is not possible with strong type due to coupling to substrate runtime. This is same issue which also lead us to duplication of most structures, so I hope we will address it at some point :/